README.md

```diff
@@ -96,6 +96,14 @@ $details->country_flag['emoji']; // 🇺🇸
 $details->country_flag['unicode']; // U+1F1FA U+1F1F8
 ```
 
+#### Country Flag URL
+
+`Details->country_flag_url` will return the link of a country's flag image.
```

Comment:
Make comment almost same as the one we have in ruby atm: https://github.com/ipinfo/ruby/pull/37#pullrequestreview-1485656602